companies/strava.json

```
@@ -0,0 +1,25 @@
+{
+    "slug": "strava",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "health"
+    ],
+    "name": "Strava, Inc.",
+    "address": "208 Utah Street\nSan Francisco, CA 94103\nUnited Staes",
```

```suggestion
    "address": "208 Utah Street\nSan Francisco\nCA 94103\nUnited States of America",
```

Thats how we format these addresses atm, compare with https://github.com/datenanfragen/data/search?q=san+francisco

---

@mal-tee In the Readme, it says however, the following
> "We want the addresses to be formatted consistently. Try to format the address according to the standards of the respective country."

According to the last sentence, the following is the [correct](https://pe.usps.com/text/pub28/28c2_001.htm) [way](https://pe.usps.com/businessmail101?ViewName=DeliveryAddress) of formatting US addresses:

JANE L MILLER
MILLER ASSOCIATES
1960 W CHELSEA AVE STE 2006R
ALLENTOWN PA 18104
UNITED STATES

> "Format the Last Line with at least one space between the city name, two–character state abbreviation, and ZIP+4 Code." - https://pe.usps.com/text/pub28/28c2_009.htm

I made the changes as requested, but we should revert back to the correct way.

---

Oh :D
@baltpeter what do you think?

---

You're right, @fm. I have opened #975 to track this.

---

Sounds good. I'll revert it on this PR and work on updating the other US addresses in the system.